ncm-metaconfig/src/main/metaconfig/irods/pan/schema.pan

```diff
@@ -0,0 +1,73 @@
+declaration template metaconfig/irods/schema;
+
+include 'pan/types';
+
+@{ type for configuring the irods_environment.json config file for clients @}
+type irods_environment_client_config = {
+    'irods_host' :  type_hostname
+    'irods_port' : type_port
+    'irods_user_name' : string
+    'irods_zone_name' : string
+    "irods_client_server_policy" : choice("CS_NEG_DONT_CARE", "CS_NEG_REFUSE", "CS_NEG_REQUIRE") = "CS_NEG_REFUSE"
+    "irods_client_server_negotiation" : choice("none", "request_server_negotiation") = "request_server_negotiation"
+    "irods_ssl_verify_server" : choice('hostname', 'cert', 'none') = 'hostname'
+    "irods_ssl_ca_certificate_file" ? string
+    "irods_encryption_algorithm": string = "AES-256-CBC"
+    "irods_encryption_key_size": long = 32
+    "irods_encryption_num_hash_rounds" : long = 16
+    "irods_encryption_salt_size" : long = 8
+};
+@{ type for configuring the irods_environment.json config file for service accounts @}
+type irods_environment_server_config = {
+    include irods_environment_client_config
+    "irods_cwd": string = "/UGent/home/rods"
+    "irods_default_hash_scheme": choice('SHA256', 'MD5') = "SHA256"
+    "irods_default_number_of_transfer_threads": long =  4
+    "irods_default_resource": string = "demoResc"
+    "irods_maximum_size_for_single_buffer_in_megabytes": long = 32
+    "irods_home": string = "/UGent/home/rods"
```

Comment:
May be better not to set a default here (not all sites have a 'UGent' home ;-) )